test/compilable/test15371.d

```diff
@@ -1,11 +1,12 @@
 module compilable.test15371;
 
-import test15371_types : type_test15371;
+import imports.test15371_types : type_test15371;
+import std.stdio: writeln;
 
 version(unittest)
 
 void main() {
     static foreach(m; __traits(allMembers, type_test15371)) {
-        __traits(getMember, m);
+        writeln(__traits(getMember, m));
```

Comment:
We don't use phobos in the test suite, use printf (just have a look at some other random tests, you'll see). 
Also the test now has output so that needs to be matched (again see other tests)

Other than that, seems fine.

Reply:
Done.